### PR TITLE
modify MongoServiceDao query wrapper to log metrics

### PR DESCRIPTION
**Pull request by moxkari**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,16 +1,23 @@
+apply plugin: 'rg-multi-project'
+
 buildscript {
     repositories {
-        jcenter()
+        //jcenter()
```
Comment (diegomon) [inline build.gradle:5]: I think you should roll back this changes because those are researchgate specific

File: build.gradle [hunk 2]
```diff
@@ -74,16 +81,18 @@ subprojects {
...
 project(":restler-core") {
-    archivesBaseName = 'restler-core'
-
-    configurations {
-        all*.exclude group: 'org.apache.solr'
-        all*.exclude group: 'org.slf4j', module: 'slf4j-log4j12'
-        all*.exclude module: 'slf4j-jdk14'
-        all*.exclude module: 'log4j'
-    }
+//    archivesBaseName = 'restler-core'
```
Comment (diegomon) [inline build.gradle:84]: Leave things commented is not nice

File: gradle.properties [hunk 3]
```diff
@@ -1 +1 @@
-version=3.0.12-SNAPSHOT
+version=3.0.12-wchang-SNAPSHOT
```
Comment (diegomon) [inline gradle.properties:1]: Don't leak this kind of version number

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -1,5 +1,6 @@
+#Thu May 28 16:47:07 CEST 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-bin.zip
-zipStoreBase=GRADLE_USER_HOME
```
Comment (diegomon) [inline gradle/wrapper/gradle-wrapper.properties:4]: Is this change related to restler?

File: restler-core/src/main/java/net/researchgate/restdsl/dao/MongoServiceDao.java [hunk 5]
```diff
@@ -42,7 +42,10 @@
 public class MongoServiceDao<V, K> implements PersistentServiceDao<V, K> {
     private static final Logger LOGGER = LoggerFactory.getLogger(MongoServiceDao.class);
 
-    private static final String QUERY_KEY = "queries.shapes.%s.%%H";
+    private static final String METRICS_SEPARATOR = "__";
```
Comment (diegomon) [inline restler-core/src/main/java/net/researchgate/restdsl/dao/MongoServiceDao.java:45]: This is very specific to our implementation of how we encode metrics but other users will use it the graphite way normally

File: build.gradle (hunk 1 above)
Comment (moxkari) [reply build.gradle:5]: got it, that makes sense 👍

**Comment by moxkari** [comment]
Closing and moving the changes up to commons-dropwizard instead
